registry/remote/repository_test.go

```diff
@@ -7082,6 +7091,7 @@ func TestRepository_pingReferrers(t *testing.T) {
 		switch {
 		case r.Method == http.MethodGet && r.URL.Path == "/v2/test/referrers/"+zeroDigest:
 			count++
+			w.Header().Set("Content-Type", ocispec.MediaTypeImageIndex)
```

Comment:
Can we have another test case for `pingReferrers` and `Referrers`, where `200` is returned with an unexpected content type?

Reply:
I added two tests since there are two places in the code that have to check the referrer's content type.